microcosm_postgres/operations.py

```diff
@@ -2,9 +2,18 @@
 Common database operations.
 
 """
+from microcosm_postgres.migrate import main
 from microcosm_postgres.models import Model
 
 
+def stamp_head(graph):
```

Comment:
I'm tempted to call `stamp_head` from `create_all` if we can automate the migrations dir lookup.